app/views/lessons/show.html.erb

```diff
@@ -7,8 +7,29 @@
 
     <main class="grid grid-cols-12 gap-x-6" data-controller="lesson-toc" data-lesson-toc-item-classes-value="no-underline hover:text-gray-800 text-sm dark:hover:text-gray-300">
       <article class="col-span-full xl:col-span-7 xl:col-start-2">
-        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do %>
-          <%= @lesson.body.html_safe %>
+        <%= render ContentContainerComponent.new(classes: 'xl:mx-0', data_attributes: { lesson_toc_target: 'lessonContent' }) do |component| %>
+        <%= @lesson.body.html_safe %>
+
+        <% component.with_footer do %>
+          <div class="py-10  xl:col-span-7 xl:col-start-2 lg:col-start-3 lg:col-span-7">
```

Comment:
Nit: Now you have the container handling all the widths and break points, we can remove most of this 💪 
We can also change to only have padding top for some breathing room, the container is giving us padding bottom.

```suggestion
          <div class="pt-10">
```